packages/safe-upgrade/tests/test_safe-upgade.lua

```diff
@@ -139,11 +139,6 @@ mtd8: 00010000 00010000 "ART"
         assert.are.same(expected, files_preserved)
     end)
 
-    it('test cmdline help', function()
```

Comment:
and we can't add lua-argparse to the image?

Reply:
Yes, we have to host it somewhere (as this package is from lime-packages).